initramfs/scripts/bootloader-script

```diff
@@ -0,0 +1,190 @@
+# -*- shell-script -*- vim:ft=sh:
```

Comment:
Correct, this gets run by adding boot=bootloader-script to the kernel command line.